### DISCUSSION TITLE
Allow multiple users and ability to "assign" tests for engineers to fix

### DISCUSSION THREAD

**Discussion by drewklein20** [sorry-cypress/sorry-cypress#871] [Ideas]
If there is currently an easy way to export test results then ignore this request.

I currently copy & paste test results to a google sheet, then assign broken tests to different engineers for fixing. In an ideal world it would be sweet to be able to select multiple tests from the results, and "assign" that selected set of tests to an engineer. They would also be able to login and see the results with the designated assigned field. Even if no authentication is setup, even just a simple string field that could be edited in bulk for selected tests would be helpful

**Comment by agoldis** [comment]
Hey @drewklein20 it is definitely a good feature, but given the recent [blocking](https://currents.dev/posts/v13-blocking) it is unlikely to be implemented in the near future because we need to do some rehaul for this project.

If you are ready to contribute - always welcome!
